packages/ceramic-cli/src/index.ts

```diff
@@ -35,6 +42,10 @@ program
   .command('show <docId> [<anchor>]')
   .description('Show the content of a document')
   .action(async (docId) => {
+    if (!validateDocId(docId)) {
+      console.error(`Invalid docId: ${docId}`)
```

Comment:
maybe exit with `process.exit(1)`

Reply:
yeah, good point. We can update this as part of #9